apps/style/JsDebuggerUi.scss

```diff
@@ -78,12 +78,6 @@ $debug-header-border-width: 1px;
   }
 }
 
-#debug-watch-header {
```

Comment:
i don't think this needs to be part of this PR, but i wonder how much work it would be to delete these styles completely and move them to `js-debugger.module.scss`. it looks like they're only imported in applab and gamelab styles

Reply:
yeah I noticed this file late when I found an errant border that remained because the new class name rule had lower precedence than the old id rule (previously everything was inline which had highest precedence). So these rules have a bit of a risk by remaining (I checked the rest and couldn't find any other conflicts). I will make a task and put it on the backlog to refactor these.

Reply:
https://codedotorg.atlassian.net/browse/SL-353